sqlglot/dialects/bigquery.py

```diff
@@ -717,7 +731,13 @@ def _parse_table_parts(
                 )
 
                 info_schema_view = f"{table_parts[-2].name}.{table_parts[-1].name}"
-                table.set("this", exp.Identifier(this=info_schema_view, quoted=True))
+                new_this = exp.Identifier(this=info_schema_view, quoted=True).update_positions(
+                    line=table_parts[-2].meta.get("line"),
+                    col=table_parts[-1].meta.get("col"),
```

Comment:
do we need start and end as well, or just line and col?

Reply:
I think we need both